### PR TITLE
*：Simplify the use of NFS backup.

### PR DESCRIPTION
**Pull request by runkecheng**
### What type of PR is this?

/documentation
/enhancement

### Which issue(s) this PR fixes?

Fixes #

### What this PR does?

Summary:

- Adjust nfs doc.
- Rename `backupFromNFS` to `NFSServerAddress`
- Merged NFS related sample file.

### CODE REVIEW

File: api/v1alpha1/backup_types.go [hunk 1]
```diff
@@ -38,9 +38,9 @@ type BackupSpec struct {
 	// If is empty, is use leader HostName
 	HostName string `json:"hostname,omitempty"`
 
-	// Represents the name of backup to NFS
+	// Represents the ip of the nfs server.
 	// +optional
-	BackupToNFS string `json:"BackupToNFS,omitempty"`
+	NFSServerAddress string `json:"NFSServerAddress,omitempty"`
```
Comment (andyli029) [inline api/v1alpha1/backup_types.go:43]: json:"NFSServerAddress
->
json:"nFSServerAddress

Comment (runkecheng) [reply api/v1alpha1/backup_types.go:43]: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L87

**Comment by acekingke** [comment]
LGTM

**Review by andyli029 — APPROVED** [review]
LGTM.

**Pull request merged**